xml/signals/PKP-PLK-2020/aspects.xml

```diff
@@ -154,18 +142,14 @@
       <name>Sygnał Ms 1</name>
       <indication>Pojedynczy ciągły sygnał niebieski</indication>
       <description>Jazda manewrowa zabroniona</description>
-      <reference></reference>
-      <comment></comment>
-      <speed></speed>
-      <speed2></speed2>
+      <speed />
+      <speed2 />
```

Comment:
Is empty speed tag valid? If not, which are the valid speeds? Maybe I should add a check for the `speed` and `speed2` in another PR?

Reply:
Empty speed tag is schema-valid.  Not sure what the code does in that case.